tasks/link-checker.js

```diff
@@ -108,6 +108,10 @@ const linkChecker = async () => {
   for (let i = 0; i < urlsToVisit.length; i++) {
     const link = urlsToVisit[i];
     let href = link.url;
+    if (href.startsWith('https://github.com/aws-amplify/docs/issues/new')) {
```

Comment:
I think we need to add another url for the "edit" links too, like `https://github.com/aws-amplify/docs/edit/main/src/pages/lib/q/platform/%5Bplatform%5D.mdx`